lib/takeaway.rb

```diff
@@ -0,0 +1,26 @@
+require 'rubygems'
+
+class Takeaway
+  attr_reader :order_klass, :order, :order_time, :menu, :credentials, :sms
+
+  def initialize(order_klass=Order, menu_klass=Menu.new.list, sms=SMS.new)
+    @order_klass = order_klass
+    @order = nil
+    @menu = menu_klass
+    @sms = sms
+  end
+
+  def view_menu
+    menu.print
+  end
+
+  def create_order(menu_klass)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `menu_klass`. If it's necessary, use `_` or `_menu_klass` as an argument name to indicate that it won't be used. You can also write as `create_order(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)